Makefile

```diff
@@ -44,8 +44,8 @@ format:
 coverage:
 	mkdir -p coverage
 	forge coverage --report lcov
-	lcov --remove lcov.info -o coverage/lcov.info 'test/*' 'script/*' --rc branch_coverage=1
```

Comment:
Should this change be here?

Reply:
You mean coverage in general? cc @jdubpark 
The change was made to ensure that the branch coverage would also show up for more users - as the previous cmds were not working for me